AgentDiscoveries-Frontend/app/sass/styles.scss

```diff
@@ -1,83 +1,83 @@
-$background-color: rgb(238, 238, 238);
-$text-color: rgb(12, 12, 12);
-$overlay-color: rgb(247, 197, 89);
-
-body {
-  background-color: $background-color;
-  color: $text-color;
-}
-
-.agent-discoveries-logo {
-  height: 100%;
-  display: inline-block;
-  padding-right: 8px;
-}
-
-.rm-3 {
-  margin-right: 0.75rem !important;
-}
-
-.ml-3 {
-  margin-left: 0.75rem !important;
-}
-
-.img-md {
-  max-width: 40%;
-  max-height: 40%;
-}
-
-.border-3 {
-  margin-top: 0.75rem;
-  margin-left: 0.75rem;
-  margin-right: 0.75rem;
-  margin-bottom: 0.75rem;
-}
-
-.overlay {
-  position: absolute;
-  top: 0;
-  bottom: 0;
-  left: 0;
-  right: 0;
-  height: 100%;
-  width: 100%;
-  opacity: 0;
-  transition: .5s ease;
-  background-color: $overlay-color;
-}
-
-.overlay-action {
-  width: 100%;
-  height: 100%;
-  
-  p {
-    color: white;
-    font-size: 20px;
-    position: absolute;
-    text-align: center;
-    top: 50%;
-    left: 50%;
-    margin-right: -50%;
-    transform: translate(-50%, -50%)
-  }
-}
-
-
-.img {
-  display: inline-block;
-  width: 100%;
-  height: 100%;
-}
-
-.profile-img-container {
-  position: relative;
-  width: 50%;
-}
-
-.profile-img-container:hover .overlay {
-  opacity: 0.8;
-}
-
-.resizeOff{
-  resize: none;
-}
+$background-color: rgb(238, 238, 238);
```

Comment:
It doesn't look like this file has changed, but it thinks that it has.. might be worth trying to remove this change from git to simplify merge life for everyone else